python/lsst/ap/association/packageAlerts.py

```diff
@@ -157,10 +203,15 @@ def run(self,
                                    objDiaForcedSources,
                                    diffImCutout,
                                    templateCutout))
-        with open(os.path.join(self.config.alertWriteLocation,
-                               f"{ccdVisitId}.avro"),
-                  "wb") as f:
-            self.alertSchema.store_alerts(f, alerts)
+
+        if self.config.produceAlertsConfig and "confluent_kafka" in sys.modules:
+            self.produceAlerts(alerts, ccdVisitId)
+
+        else:
```

Comment:
I'd prefer that we use separate configs for producing and writing alerts: `doProduceAlerts` and `doWriteAlerts`.  Then we can raise an error if `doProduceAlerts` is set and confluent_kafka isn't loaded.

Reply:
As mentioned above, we now have both, as well as a check if neither is called set since that would be a situation where nothing happens to the alerts.